stdlib/EvaluatorSettings.pkl

```diff
@@ -67,6 +67,9 @@ allowedModules: Listing<String(isRegex)>?
 /// ```
 allowedResources: Listing<String(isRegex)>?
 
+/// Whether errors should be formatted with ANSI color.
+color: ("never"|*"auto"|"always")?
```

Comment:
Nullable _plus_ default here doesn't do anything, does it?

```suggestion
/// When to format messages with ANSI color codes.
///
/// Possible values:
///
/// - `"never"`: Never format 
/// - `"auto"`: Format if the process' stdin, stdout, or stderr are connected to a console.
/// - `"always"`: Always format
@Since { version = "0.27.0" }
color: ("never"|"auto"|"always")?
```

Reply:
I felt it was a reasonable way to make it self-documenting that `"auto"` was the default setting, but sure; it's not functional.